analyzer/vulnerability-consumer/src/main/java/eu/fasten/analyzer/vulnerabilityconsumer/utils/exceptions/CallablesNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package eu.fasten.analyzer.vulnerabilityconsumer.utils.exceptions;
+
+/**
+ * Exception thrown if patches are present but no fasten_uri is found for the vulnerability.
+ */
+public class CallablesNotFoundException extends Exception {
+    /**
+     * Default constructor without a message.
+     */
+    public CallablesNotFoundException() {
```

Comment:
I would suggest to add a little bit of context to this exception such as information about the callable that we are searching for.

Same apply to other custom Exception classes.